lua/xcodebuild/init.lua

```diff
@@ -314,6 +314,7 @@ function M.setup(options)
   vim.api.nvim_create_user_command("XcodebuildOpenInXcode", call(actions.open_in_xcode), { nargs = 0 })
   vim.api.nvim_create_user_command("XcodebuildQuickfixLine", call(actions.quickfix_line), { nargs = 0 })
   vim.api.nvim_create_user_command("XcodebuildCodeActions", call(actions.show_code_actions), { nargs = 0 })
+  vim.api.nvim_create_user_command("XcodebuildToggleBinValidation", call(actions.toggle_bin_validation), { nargs = 0 })
```

Comment:
I would change this user command to accept a bool parameter, it will be easier for the user to use and more clear what's the current state. So `:XcodebuildSetBinValidation true`.